gulp-tasks/workbox:generate-contributors.js

```diff
@@ -0,0 +1,94 @@
+const GitHubApi = require('github');
```

Comment:
some filesystems may puke on a filename with a colon in it, can we change the colon?